requirements/constraints.txt

```diff
@@ -15,4 +15,4 @@ futures ; python_version == "2.7"
 
 # latest botocore release cap upper version of python-dateutil to 2.8.0 and that is not
 # compatible with other packages. That's why adding a cap here to avoid boto latest release untill they fix the issue.
```

Comment:
Should update the comment to reflect what's being pinned.  Preferably also link to https://github.com/boto/botocore/issues/1872 to make it easier to track when this is fixed upstream.